Sources/NIO/SocketChannel.swift

```diff
@@ -49,7 +49,7 @@ class BaseSocketChannel<T: BaseSocket>: SelectableChannel, ChannelCore {
     internal let socket: T
     private let closePromise: EventLoopPromise<Void>
     private let selectableEventLoop: SelectableEventLoop
-    private let addressesCached: AtomicBox<Box<(local:SocketAddress?, remote:SocketAddress?)>> = AtomicBox(value: Box((local: nil, remote: nil)))
+    private let addressesCached: AtomicBox<Box<(local: SocketAddress?, remote: SocketAddress?)>> = AtomicBox(value: Box((local: nil, remote: nil)))
```

Comment:
Forgot to mention I also ran `swiftlint autocorrect` on the files I modified (to clean up all the random whitespace Xcode likes to insert) which is the cause of the nit-picky changes like this.

Reply:
@toffaletti I wonder if we should better exclude the style changes to make merging easier and keep the PR focused on only the "feature". WDYT ?

Reply:
Yeah, I think that would be much nicer. :smile: